.cbmc-batch/include/proof_helpers/proof_allocators.h

```diff
@@ -14,12 +14,14 @@
  */
 
 #pragma once
+
 #include <aws/common/common.h>
+#include <stdlib.h>
 
 /**
- * Use 1GB limit for malloc in order to avoid spurious pointer offsets
+ * Use 32GB limit for malloc in order to avoid spurious pointer offsets
  */
-#define MAX_MALLOC 1073741824
+#define MAX_MALLOC 34359738368
```

Comment:
Why is 32GB the right number (also, I'd spell out the multiplication, possibly as a shift)?